docs/installation.rst

```diff
@@ -151,5 +151,5 @@ Now start your server, visit your admin pages (e.g. http://localhost:8000/admin/
 and follow these steps:
 
 1. Add a ``Site`` for your domain, matching ``settings.SITE_ID`` (``django.contrib.sites`` app).
-2. For each OAuth based provider, add a ``Social App`` (``socialaccount`` app).
+2. For each OAuth based provider, add a ``Social applications`` (inside the ``socialaccount`` app).
```

Comment:
This should be singular (application).

Reply:
Just trying to match what the user sees 
<img width="650" alt="Screen Shot 2019-04-25 at 6 03 57 PM" src="https://user-images.githubusercontent.com/587438/56771422-9a162f80-6784-11e9-81c9-c546fb222038.png">

Could rephrase it to `For each OAuth based provider, add a new instance under ``Social applications``.`

Reply:
It does not state "Add social applications" on screen. On screen, there is a link to all applications (plural), and an "Add" link, which adds a single application.